ui/css/style.css

```diff
@@ -99,6 +99,9 @@ a {
 .content-margin {
```

Comment:
@NkFab The ui/css/style.css has to be put in .gitignore to avoid conflicts with other developers because the whole file is being rebuilt everything that you compile sass' files

Reply:
This file is already added in the .gitignore file I don't know why it's not ignoring it. Let me check the reason for that.